scripts/test/bulkImportPerformance/system-test-instance.properties

```diff
@@ -49,7 +49,7 @@ sleeper.bulk.import.eks.repo=changeme
 sleeper.vpc=changeme
 
 # The subnet to deploy tasks to
```

Comment:
Does this comment need updating to "A comma separated list of subnets to deploy ECS tasks to."?